### ISSUE TITLE
Windows installer doesn't update on first go

### ISSUE THREAD

**Issue by askmeaboutlo0m** [drawpile/Drawpile#1201]
Recently, the Windows installer for the continuous release stopped updating over the old installation. Trying to "repair" the installation makes it work though.

It's not totally clear what happens on the first attempt of installing it. Either nothing at all or maybe it installs it into %APPDATA%.

Also no clue why this suddenly started happening, since we didn't change anything about the installer input scripts. Possibly a Windows update sabotaging us, or GitHub updated the WiX toolkit to a new version on their Actions boxes that ends up generating a bad installer.

Installer logs can be generated by running it via cmd:

```
msiexec /i "C:\Path\To\Drawpile-x.y.z.msi" /L*V "C:\Path\To\LogFile.txt"
```

Credit Bluestrings, Blozzom and, uh, anonymous in the changelog for reporting this.

**Comment by MorrowShore** [comment]
Could not reproduce issue.

First installed https://github.com/drawpile/Drawpile/releases/download/2.2.1/Drawpile-2.2.1-64bit.msi
and then: https://github.com/drawpile/Drawpile/releases/download/continuous/Drawpile-2.2.2-beta.1-108-g0b1469fca-64bit.msi

Both installed to D:\Drawpile, on top of each other.

Ran the program after each installation.

![image](https://github.com/drawpile/Drawpile/assets/97566823/2e867ab0-c998-4681-a151-a1d36a55da62)
![image](https://github.com/drawpile/Drawpile/assets/97566823/0f108e74-18cf-450f-9611-41109221d934)

No issue was found.

**Comment by amoruwu** [comment]
I believe, that this might be "fixed". The installer just forgot the location of Drawpile (for whatever reason) so went to the default I believe which is %APPDATA%. People didn't see/didn't notice that the the location changed so were confused.

**Comment by amoruwu** [comment]
Ah just checked downloading drawpile. It went to C:\Program Files\Drawpile\ not %APPDATA%.

**Comment by askmeaboutlo0m** [comment]
I happened to boot into Windows earlier, so I messed around with this. Installing 2.2.1 first and then installing the continuous release I ended up with *two* entries for Drawpile in the Add/Remove Programs control panel and also two shortcuts on the desktop.

Installing 2.2.1 first and then 2.2.2-beta.1 afterwards didn't cause this, it overwrote the old version like it should have. Installing continuous afterwards gave me another entry again, so it's not related to the version jump from 2.2.1 to 2.2.2.

Something must have changed between 2.2.1 and now, either in our installer or in the WiX toolage that's in the GitHub Actions. I'll look into it further when I get the time.

**Comment by askmeaboutlo0m** [comment]
It happens even when going back to the commit from 2.2.2-beta.1, so it looks like it's nothing that changed in Drawpile's code or anything. I kind of suspected that it had to do with the version number somehow, but even changing that around doesn't help at all.

So it looks like it's nothing that changed about our code, but instead just the WiX toolkit breaking on its own. Which is pretty cool.

I gathered some logs and I guess I'll have to pore over those to figure out why it suddenly thinks it's installing a different program instead of updating an existing one.

**Comment by askmeaboutlo0m** [comment]
Fixed, see rant in 8c37b9dd2a73541e7557bb2ec115142031eb34fb
